lib/flexirest/caching.rb

```diff
@@ -4,7 +4,7 @@ module ClassMethods
       @@perform_caching = true
 
       def perform_caching(value = nil)
-        @perform_caching ||= nil
+        @perform_caching = nil unless defined?(@perform_caching)
```

Comment:
Just a nitpick; but, shouldn't this be `instance_variable_defined?(:@perform_caching)`?